backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/timebooking/S_TimeBooking.java

```diff
@@ -93,21 +99,25 @@ public void recomputeIssueProgress(@NonNull final I_S_TimeBooking record)
 
 		if (record.getS_Issue_ID() != oldRecord.getS_Issue_ID())
 		{
+			final Effort bookedEffort = Effort.of(record.getBookedSeconds().longValue());
```

Comment:
I would rename Effort.of(long)  method to Effort.ofSeconds.
Reason: 
* much more descriptive
* i have the feeling soon we will introduce ofHours(int) method too.